CAIPs/caip-25.md

```diff
@@ -185,30 +162,39 @@ An example of a successful response follows:
         "notifications": ["accountsChanged", "chainChanged"],
         "accounts":["eip155:42161:0x0910e12C68d02B561a34569E1367c9AAb42bd810"]
       },
-      "wallet": {
+      "eip155:0": {
         "methods": ["wallet_getPermissions", "wallet_creds_store", "wallet_creds_verify", "wallet_creds_issue", "wallet_creds_present"],
         "notifications": []
       },
       "cosmos": {
         ...
       }
     },      
+    "scopedProperties": {
+      "eip155:42161": {
+        "rpcDocuments": "https://example.com/wallet_extension.json",
```

Comment:
is this getting moved from CAIP-217? If not I think its a bit confusing to use it as an example? @bumblefudge

Reply:
oooh, good catch, that was unintentional!